sdk/src/test/java/io/opentelemetry/sdk/resources/EnvVarResourceTest.java

```diff
@@ -67,4 +68,15 @@ public void parseResourceAttributes_quotes() {
     Attributes result = EnvVarResource.parseResourceAttributes("value=\"foo\"");
     assertThat(result).isEqualTo(Attributes.of("value", stringAttributeValue("foo")));
   }
+
+  @BeforeClass
+  public static void init() {
+    System.setProperty("otel.resource.attributes", "value = foo");
```

Comment:
We should have this set right in the test itself (the rest of the tests don't need this part).

Reply:
Hi~ This is because I put this static final field. The following code：
```
private static final Resource ENV_VAR_RESOURCE =
      new Builder().readEnvironmentVariables().readSystemProperties().build();
```
(it may inited before)

Reply:
Hmm, isn't that true if the class happens to be loaded before this test is run? It seems like a lucky coincidence that isn't currently happening in the build.

I think to make this test robust we should stick with calling `new Builder().readEnvironmentVariables().readSystemProperties().build();` in the test instead of relying on the static instance being loaded after the test.

Also, make sure you reset the value of the property after the test is complete.

Reply:
agree. fixed at lastest commit. Thanks.